compat-tool/compat.py

```diff
@@ -19,49 +19,49 @@
 
 def double_check(checkOperator, checkLine, checkLineLength):
     foundOperator = False
-    
+
```

Comment:
Please send a separate PR that just strips all trailing whitespaces to make future PRs smaller